dts/st/f3/stm32f301c(6-8)tx-pinctrl.dtsi

```diff
@@ -686,10 +686,22 @@
 				pinmux = <STM32_PINMUX('A', 0, AF3)>;
 			};
 
+			/omit-if-no-ref/ tsc_g1_io1_pa0: tsc_g1_io1_pa0 {
+				pinmux = <STM32_PINMUX('A', 0, AF3)>;
+			};
+
```

Comment:
All these tsc pins are duplicated. Any ideas why? Does it come from the XML source or is it a problem with our script?

Reply:
It seems that TSC filer is present twice.... This should be fixed